docs_guide/sphinx_guide/how_to_add_sections.rst

```diff
@@ -0,0 +1,96 @@
+####################################
+How to add sections to documentation
+####################################
+
+.. include:: ../others/how-to_type.rst
+
+This guide shows how to separate your documents in parts. 
+
+.. include:: ../others/general_pre-requisites.rst
+
+You can organize your documents into:
+
+* Sections.
+* Subsections.
+* Subsubsections.
+* Paragraphs.
+
+Create sections
+===============
+
+In order to create a new section you can write the name of the section covered below by equal signs ``=``. For example,
```

Comment:
You left the "covered below by `=`" sentence here (and many times along this guide)

However, instead of removing it entirely, I suggest using the phrasing from [Sphinx's official documentation](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections):

>Section headers ([ref](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections)) are created by underlining (and optionally overlining) the section title with a punctuation character, at least as long as the text:

In short, I'd change "covered below by ..." to "underlined by ..."

Reply:
I am using the same wording as the sphinx official documentation.